routes/api.php

```diff
@@ -1365,5 +1303,44 @@
         ])->name('api.assets.labels');
         // end generate label routes
 
+    /**
+     * Uploaded files API routes
+     */
+
+    // List files
+    Route::get('{object_type}/{id}/files',
+        [
+            Api\UploadedFilesController::class,
+            'index'
+        ]
+    )->name('api.files.index')
+        ->where(['object_type' => 'assets|models|users|locations|accessories|consumables|licenses|components']);
```

Comment:
LOVE this! It answers the question I was pondering of “what if the `$object_type` was somehow not one of the ones that we have listed here…”